appveyor.yml

```diff
@@ -72,10 +72,10 @@ install:
           else
             {$root = "C:\Miniconda-x64"}
           $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    - cmd: conda config --set show_channel_urls true
     - cmd: conda update --yes --quiet conda
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls yes
```

Comment:
Sure, I understand. It is useful for debugging. So we really should have that info anywhere we can as you suggested. Hence these PRs.